gcc/config/i386/cygming.h

```diff
@@ -458,7 +458,7 @@ do {						\
 #define SUBTARGET_MANGLE_DECL_ASSEMBLER_NAME i386_pe_mangle_decl_assembler_name
 
 #undef TARGET_ASM_ASSEMBLE_VISIBILITY
-#define TARGET_ASM_ASSEMBLE_VISIBILITY i386_pe_assemble_visibility
+#define TARGET_ASM_ASSEMBLE_VISIBILITY mingw_pe_assemble_visibility
```

Comment:
Do we still need this file in our repository? Isn't the right time to remove it (in a separate PR)?

Reply:
are you talking about winnt.cc?

Reply:
No, I was rather thinking about merging the common parts of `gcc/config/i386/cygming.h` and `gcc/config/aarch/cygming.h` to `gcc/config/mingw/cygming.h`

Reply:
It is still quite different. It makes sense to try it after implementing SEH.